### PR TITLE
feat(readme): Add README.md for chart package

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

### CODE REVIEW

**Comment by jbrundage** [comment]
@GordonSmith https://github.com/jbrundage/Visualization/tree/readme_b31/packages/chart

Can we add the images after each package has a README.md? I'd like to work on the ability to generate images of the gallery samples using phantomjs.

File: packages/chart/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        </script>
+    </body>
+</html>
+```
+
+## Getting Started with @hpccjs
+* _[Quick Start](https://github.com/hpcc-systems/Visualization/wiki/Quick-Start)_
+* _[Tutorials](https://github.com/hpcc-systems/Visualization/wiki/Tutorials)_
+* _[Gallery](https://raw.githack.com/hpcc-systems/Visualization/master/demos/gallery/gallery.html)_ ([alt](https://rawgit.com/hpcc-systems/Visualization/master/demos/gallery/gallery.html))
+* _[Wiki](https://github.com/hpcc-systems/Visualization/wiki)_
```
[the remark this answers is not in the document]
Comment (jbrundage) [reply packages/chart/README.md]: But that is secondary info relative to the framework as a whole. Maybe we could just move the Quick Start to the top?

[the remark this answers is not in the document]
Comment (GordonSmith) [reply packages/chart/README.md]: How about:
```
# @hpcc-js/chart
This package is part of the mono repository "@hpcc-js" (aka Visualization Framework), for more information including [Quick Start](https://github.com/hpcc-systems/Visualization/wiki/Quick-Start), [Gallery](https://raw.githack.com/hpcc-systems/Visualization/master/demos/gallery/gallery.html) and [Tutorials](https://github.com/hpcc-systems/Visualization/wiki/Tutorials), please visit the main page on GitHub:  [hpcc-systems/Visualization](https://github.com/hpcc-systems/Visualization).
```

[the remark this answers is not in the document]
Comment (GordonSmith) [reply packages/chart/README.md]: Then there would be no need for the "Getting Started" section?

My main concern is that once you add some screen shots, then the "Getting Started" will end up pages down...

**Pull request merged**